### ISSUE TITLE
Ant version change is not taking effect

### ISSUE THREAD

**Issue by tsdrahangdale** [cedx/setup-ant#6]
Hello ,

Looks like the github action is unable to change the Ant version specified in the workflow

![image](https://github.com/user-attachments/assets/a7041279-b91b-4ef3-90da-70fc595f19a8)

in one case it did not return error but also version specified was not set in the file system 
![image](https://github.com/user-attachments/assets/ad02e97e-508b-48ce-9722-4837ac986539)

**Comment by cedx** [comment]
This is because Apache only provides a limited set of binaries.

See: https://dlcdn.apache.org/ant/binaries/
And: https://github.com/cedx/setup-ant/blob/main/src/data.coffee

So you can't use a version prior to v1.9.16 😞

**Comment by cedx** [comment]
Hmmm I just found this archive:
https://archive.apache.org/dist/ant/binaries/

I guess I can use it to support older releases...

**Comment by cedx** [comment]
@tsdrahangdale I've made a few changes. Can you test the `main` branch/last commit?

```yaml
- uses: cedx/setup-ant@main
  with:
    version: 1.8.1 # or whatever you want
```

**Comment by tsdrahangdale** [comment]
@cedx thanks a ton for fixing this. Tested this and it works now. You are lifesaver 🙏🏻🤩

**Comment by cedx** [comment]
I've released the new version v4.1.0 and updated the `v4` tag accordingly.

```yaml
uses: cedx/setup-ant@v4
```
